src/main/scala/io/radicalbit/nsdb/connector/kafka/sink/conf/NSDbConfigs.scala

```diff
@@ -63,9 +63,31 @@ object NSDbConfigs {
   val NSDB_AT_LEAST_ONCE_RETRIES_DEFAULT = 10
 
   val NSDB_AT_LEAST_ONCE_RETRY_INTERVAL         = "nsdb.at.least.once.retry.interval"
-  val NSDB_AT_LEAST_ONCE_RETRY_INTERVAL_DOC     = "Time to sleep from a retry to anther when AT_LEAST_ONCE semantic is set"
+  val NSDB_AT_LEAST_ONCE_RETRY_INTERVAL_DOC     = "Time to sleep from a retry to another when AT_LEAST_ONCE semantic is set"
   val NSDB_AT_LEAST_ONCE_RETRY_INTERVAL_DEFAULT = "500 milliseconds"
 
+  val NSDB_MAPPING_METRICS = "nsdb.mapping.metrics"
+  val NSDB_MAPPING_METRICS_DOC =
+    "Metrics value for mapping configuration in dotted notation (mutually exclusive with nsdb.kcql)"
+
+  val NSDB_MAPPING_VALUES     = "nsdb.mapping.values"
+  val NSDB_MAPPING_VALUES_DOC = "Values value for mapping configuration in dotted notation (optional)"
+
+  val NSDB_MAPPING_TAGS     = "nsdb.mapping.tags"
+  val NSDB_MAPPING_TAGS_DOC = "Tags value for mapping configuration in dotted notation (optional)"
+
+  val NSDB_MAPPING_TIMESTAMPS     = "nsdb.mapping.timestamps"
+  val NSDB_MAPPING_TIMESTAMPS_DOC = "Timestamps value for mapping configuration in dotted notation (optional)"
+
+  // inner property
```

Comment:
we should avoid the inner word, to me at least it's misleading. I guess we can just remove the word inner and everything will be more clear. It's my opinion, others are welcome